### PR TITLE
fix(docs) Update supported languages in README

### LINKED ISSUE #2505: Add Python to the list of supported languages in the README
**Describe the bug**
The following languages are currently supported:
- Java
- JavaScript, TypeScript
- .NET

**To Reproduce**
https://github.com/awslabs/aws-cdk/blob/master/README.md

**Expected behavior**
- Java
- JavaScript, TypeScript
- .NET
- Python

### PR DESCRIPTION
**Pull request by ScottBrenner**
Updating the list of supported languages in the README based on https://docs.aws.amazon.com/cdk/latest/guide/what-is.html

> Use the CDK to define your cloud resources using one of the supported programming languages: **C#/.NET, Java, JavaScript, Python, or TypeScript.**

Python should be here, at least.

Fixes #2505.

----

### Pull Request Checklist

* [ ] Testing
  - Unit test added (prefer not to modify an existing test, otherwise, it's probably a breaking change)
  - __CLI change?:__ coordinate update of integration tests with team
  - __cdk-init template change?:__ coordinated update of integration tests with team
* [ ] Docs
  - __jsdocs__: All public APIs documented
  - __README__: README and/or documentation topic updated
  - __Design__: For significant features, design document added to `design` folder
* [ ] Title and Description
  - __Change type__: title prefixed with **fix**, **feat** and module name in parens, which will appear in changelog
  - __Title__: use lower-case and doesn't end with a period
  - __Breaking?__: last paragraph: "BREAKING CHANGE: <describe what changed + link for details>"
  - __Issues__: Indicate issues fixed via: "**Fixes #xxx**" or "**Closes #xxx**"
* [ ] Sensitive Modules (requires 2 PR approvers)
  - IAM Policy Document (in @aws-cdk/aws-iam)
  - EC2 Security Groups and ACLs (in @aws-cdk/aws-ec2)
  - Grant APIs (only if not based on official documentation with a reference)

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

**Comment by eladb** [comment]
Please indicate that this fixe #2505 (thanks!)

**Comment by ScottBrenner** [comment]
Sure, added!

File: README.md [hunk 1]
```diff
@@ -19,9 +19,10 @@ and efficient manner.
```
[the remark this answers is not in the document]
Comment (ScottBrenner) [reply README.md]: It looks like the logos are not loading for me, are you able to view them?
image
Seems the path `docs/src/_images/` does not exist

**Comment by rix0rrr** [comment]
Elad took care of this in the mean time. Thanks for pointing it out!

**Pull request closed without merging**